spicepy/_client.py

```diff
@@ -56,7 +58,7 @@ def __init__(
 
     def _authenticate(self):
         headers = [self._flight_client.authenticate_basic_token("", self._api_key)]
-        self._flight_options = flight.FlightCallOptions(headers=headers)
+        self._flight_options = flight.FlightCallOptions(headers=headers, timeout=DEFAULT_QUERY_TIMEOUT_SECS)
 
     def query(self, query: str) -> flight.FlightStreamReader:
```

Comment:
Need to be able to pass in a custom optional timeout

Reply:
Done